wagtail/wagtailadmin/static_src/wagtailadmin/scss/components/_main-nav.scss

```diff
@@ -32,7 +32,7 @@ $footer-submenu: $submenu-color;
 }
 
 .nav-main {
-    top: 43px;
+    top: 53px;
```

Comment:
This does not work, because on the mobile breakpoint `.nav-main` is set to `position: static;` (the default value).